gyp/pylib/gyp/generator/make.py

```diff
@@ -1193,6 +1193,46 @@ def WriteMacInfoPlist(self, bundle_deps):
                     part_of_all=True)
     bundle_deps.append(out)
 
+  def MacCFlags(self, xcodeFlags, userFlags):
```

Comment:
Python style for variable names (as generally used in the rest of this file) is snake case (xcode_flags) instead of camel case (xcodeFlags).